src/blog/2019-02-13/announcing-stryker-1-0.pug

```diff
@@ -0,0 +1,113 @@
+            If Stryker runs without deprecation warnings, you're ready to upgrade
+
+        3. **Update your global stryker-cli**
+
+            If you've installed the `stryker-cli`, please update that one as well: `npm i -g stryker-cli`
+
+        4. **Upgrade your dependencies**
+
+            Update your package.json file. 
+
+            * `stryker` ➡ `@stryker-mutator/core`
+            * `stryker-api` ➡ _none, this can be removed_
+            * `stryker-babel-transpiler` ➡ `@stryker-mutator/babel-transpiler`
+            * `grunt-stryker` ➡ `@stryker-mutator/grunt-stryker`
+            * `stryker-html-reporter` ➡ `@stryker-mutator/html-reporter`
+            * `stryker-jasmine` ➡ `@stryker-mutator/jasmine-framework`
+            * `stryker-jasmine-runner` ➡ `@stryker-mutator/jasmine-runner`
+            * `stryker-javascript-mutator` ➡ `@stryker-mutator/javascript-mutator`
+            * `stryker-jest-runner` ➡ `@stryker-mutator/jest-runner`
+            * `stryker-karma-runner` ➡ `@stryker-mutator/karma-runner`
+            * `stryker-mocha-framework` ➡ `@stryker-mutator/mocha-framework`
+            * `stryker-mocha-runner` ➡ `@stryker-mutator/mocha-runner`
+            * `stryker-mutator-specification` ➡ `@stryker-mutator/mutator-specification`
+            * `stryker-test-helpers` ➡ `@stryker-mutator/test-helpers`
+            * `stryker-typescript` ➡ `@stryker-mutator/typescript`
+            * `stryker-util` ➡ `@stryker-mutator/util`
+            * `stryker-vue-mutator` ➡ `@stryker-mutator/vue-mutator`
+            * `stryker-wct-runner` ➡ `@stryker-mutator/wct-runner`
+            * `stryker-webpack-transpiler` ➡ `@stryker-mutator/webpack-transpiler`
+
+            The versioning of these dependencies start at 1.0.0, so feel free to use `^1.0.0` to specify the version.
+        
+            After updating your package.json, you'll need to run `npm install` (or `yarn install`) once more. 
+
+        5. **Take it for a spin**
+            
+            Stryker should now work as expected. Try it out with: `npx stryker run`.
+
+        ## Release cycle 
+
+        From now on, we adhere to strict [semantic versioning](https://semver.org/spec/v2.0.0.html), or semver for short.
+        This means that it's easy to distinguish between bug fixes, new features or breaking changes just 
+        by looking at the version number. 
+
+        Futhermore, we've decided to use _fixed versioning_ for our packages, meaning that all `@stryker-mutator/*` packages
+        will _have the same version number_. For example, if your local `@stryker-mutator/core` version is `1.0.0`, your
+        `@stryker-mutator/html-reporter` should also be at that exact version. This makes maintaining your Stryker dependencies a breeze. 
+        
+        This is comparable to how Angular or React do their versioning.
+
+        ## Release schedule
```

Comment:
This feels more like a 'What's next' to me than a 'release schedule'. Especially as it's (almost) the same as before. Maybe 'what's next' would be clearer, and also gives the opportunity to talk about new planned features/supported frameworks, etc (if any).